web/app/common/components/status-icon/status-icon.component.html

```diff
@@ -0,0 +1,14 @@
+<ng-container [ngSwitch]="status">
+  <mat-icon *ngSwitchCase="BuildStatus.SUCCESS" class="fci-status-icon-success">
+    sentiment_very_satisfied
+  </mat-icon>
+  <mat-icon *ngSwitchCase="BuildStatus.PENDING" class="fci-status-icon-pending">
+    sentiment_neutral
```

Comment:
`sentiment_unknown` might be better?

Reply:
ehh, maybe not. Yo no se

Reply:
These are all the strings provide by the material icon set https://material.io/icons/

`sentiment_unknown` isn't a provide icon.